### PR TITLE
Multi bls secpr1 more crypto

### PR DESCRIPTION
**Pull request by sasurobert**
See:
 - https://github.com/multiversx/mx-vm-executor-rs/pull/38

SHA256 of Wasmer2 libraries, as on 2024-04-12:

```
984bbfc0c710ddea71f472732be95077925110e5d42280a00f9a4ef29819b4fb  ./libvmexeccapi.so
0135f0405bd35dbbc27357dca5850ab337ee2196d0b97c71ce2bf4069764122f  ./libvmexeccapi.dylib
11d6d27e7352f918a2369cad5c12c4398a2791c2ee00a21487a6fb0575e2c44a  ./libvmexeccapi_arm.so 
b0702f91550d912735ab1d6ec2a24db1349ee8979731ce4d3c2441b5a4ebc9c6  ./libvmexeccapi_arm.dylib
```

### CODE REVIEW

File: scenario/gasSchedules/gasScheduleV4.toml [hunk 1]
```diff
@@ -225,6 +225,9 @@
     UnmarshalCompressedECC = 270000
     GenerateKeyECC = 7000000
     EncodeDERSig = 10000000
+VerifySecp256r1 = 2000000
```
Comment (ccorcoveanu) [inline scenario/gasSchedules/gasScheduleV4.toml:228]: Alignement issue ^
+ are the values ok to be equal for different types of verifications?

File: vmhost/hostCore/host.go [hunk 2]
```diff
@@ -580,31 +572,6 @@ func (host *vmHost) EpochConfirmed(epoch uint32, _ uint64) {
 	}
 }
 
-// FixOOGReturnCodeEnabled returns true if the corresponding flag is set
-func (host *vmHost) FixOOGReturnCodeEnabled() bool {
```
Comment (ccorcoveanu) [inline vmhost/hostCore/host.go:584]: q: why are all these flags removed?

**Review by andrei-marinica — APPROVED** [review]

File: vmhost/hostCore/host.go (hunk 2 above)
Comment (sasurobert) [reply vmhost/hostCore/host.go:584]: Not needed. Not used, should have been cleaned before.

Comment (ccorcoveanu) [reply vmhost/hostCore/host.go:584]: ok, so unrelated with this PR, just some extra cleanup, got it 👍

File: scenario/gasSchedules/gasScheduleV4.toml (hunk 1 above)
Comment (ccorcoveanu) [reply scenario/gasSchedules/gasScheduleV4.toml:228]: this still seems to be here

**Review by andrei-marinica — APPROVED** [review]

**Review by andrei-marinica — APPROVED** [review]

File: scenario/gasSchedules/gasScheduleV4.toml (hunk 1 above)
Comment (sasurobert) [reply scenario/gasSchedules/gasScheduleV4.toml:233]: done

**Review by ccorcoveanu — DISMISSED** [review]

**Review by ccorcoveanu — APPROVED** [review]

**Review by laurci — APPROVED** [review]

**Review by andrei-marinica — APPROVED** [review]

**Pull request merged**